ui/src/app/pages/workspace/workspace-about.tsx

```diff
@@ -178,20 +185,25 @@ export const WorkspaceAbout = fp.flow(
       this.loadUserRoles(workspace);
     }
 
-    async loadInitialCreditsUsage(workspace: WorkspaceData) {
```

Comment:
does not need to be async: the `onSuccess` callback can execute in another context